src/auth/auth.controller.ts

```diff
@@ -18,32 +18,47 @@ import { UserGuard } from '../common/guards/user.guard';
 import { VerificationSerivce } from './verification.service';
 import { User } from '@prisma/client';
 
-@ApiTags('AUTH')
+@ApiTags('auth')
 @Controller('auth')
 export class AuthController {
   constructor(
     private readonly authService: AuthService,
     private readonly verificationService: VerificationSerivce
   ) {}
 
-  @ApiOperation({ summary: 'SIGN UP USER' })
-  @ApiResponse({ status: 201, description: 'link sent' })
+  @ApiOperation({ summary: 'Sign up user' })
+  @ApiResponse({
+    status: 201,
+    description: `{
+    success: true,
+    message: 'User created successfully, please check your email to verify your account',
+  }`,
+  })
```

Comment:
You are not supposed to do this, instead need to provide a schema. 

Here is the example. 

```js
  @ApiOperation({ summary: 'Sign up user' })
  @ApiResponse({
    status: 201,
    description: 'User created succesfully',
    schema: {
      properties: {
        success: { type: 'boolean' },
        message: {
          type: 'string',
        },
      },
    },
  })
```